src/App/index.js

```diff
@@ -1,13 +1,23 @@
 
-import React, {Component} from "react";
-import style from "./style.scss";
+import React, {Component, PropTypes} from 'react';
+import {Provider} from 'react-redux';
+import style from './style.scss';
+
+import store from './store';
 
 export default class App extends Component {
+    static propTypes = {
```

Comment:
You do actually need to declare children propTypes: https://github.com/yannickcr/eslint-plugin-react/issues/7. If you meant that App doesn't need children, it does because the whole point is to wrap other components with the provider and base styles.